Description: Add org.eclipse.jgit.pgm to jgit console classpath
Author: Jakub Adam <jakub.adam@ktknet.cz>
--- a/org.eclipse.jgit.pgm/jgit.sh
+++ b/org.eclipse.jgit.pgm/jgit.sh
@@ -90,7 +90,7 @@
 	;;
 esac
 
-CLASSPATH="$cp"
+CLASSPATH="$cp:/usr/share/java/org.eclipse.jgit.pgm.jar"
 export CLASSPATH
 
 java=java
